homeassistant/components/sonos/strings.json

```
@@ -50,6 +50,18 @@
         "name": "Music surround level"
       }
     },
+    "select": {
+      "dialog_level": {
+        "name": "Dialog level",
+        "state": {
+          "off": "Off",
+          "low": "Low",
+          "medium": "Medium",
+          "high": "High",
+          "max": "Max"
```

We can use some common states here:

```suggestion
          "off": "[%key:common::state::off%]",
          "low": "[%key:common::state::low%]",
          "medium": "[%key:common::state::medium%]",
          "high": "[%key:common::state::high%]",
          "max": "Max"
```